### ISSUE TITLE
Get rid of GMS dependency

### ISSUE THREAD

**Issue by 12people** [AmolGangadhare/flutter_barcode_scanner#299]
**Is your feature request related to a problem? Please describe.**
Not all Android devices come with Google Services support and some open-source app repositories strictly prohibit integrating proprietary packages like it.

**Describe the solution you'd like**
It'd be great if this package could get rid of its dependency on Google Play services — specifically `'com.google.android.gms:play-services-vision`.

**Comment by BjWe** [comment]
i don't think it can be done that easily. the camera handling, logic of detecting bar- and qr-codes is in that package. removing this dependency is like taking the engine out of the car.

**Comment by cracket** [comment]
@BjWe to help your imagination grow - see plenty of other cars without engine which work very well scanning QR here:
[https://search.f-droid.org/?q=qr&lang=en](url)
